packages/theme/src/themes.ts

```diff
@@ -136,8 +138,10 @@ export const darkTheme = createTheme({
       info: '#2E77D0',
       error: '#E22134',
       text: '#FFFFFF',
+      textDark: '#424242',
```

Comment:
These should probably be suffixed with `*Contrast` instead, otherwise the dark theme would logically need "textLight" 😅